meta.nim

```diff
@@ -0,0 +1,17 @@
+## macro utility
```

Comment:
1. This should probably be in `futils`, I'd rather not have another small file here.
2. Aren't the check functions very similar to [expectKind](https://nim-lang.org/docs/macros.html#expectKind%2CNimNode%2CNimNodeKind), without the condition?

Reply:
afaik you cannot use custom message with `expect{Kind|Len}`